engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/BuiltinTypesTest.scala

```diff
@@ -319,7 +319,7 @@ class BuiltinTypesTest
       3
     ) should contain theSameElementsAs Seq(
       Api.Response(requestId, Api.PushContextResponse(contextId)),
-      TestMessages.update(contextId, idMain, "Enso_Test.Test.Main.Foo.Bar"),
+      TestMessages.update(contextId, idMain, ConstantsGen.FUNCTION_BUILTIN),
```

Comment:
This might seem like a weird change at first sight but consider that `Bar` is a constructor taking an argument. So it's a function.